### PR TITLE
Add smtcoq_trakt.

### PR DESCRIPTION
**Pull request by tomaz1502**
We are adding a new version of SMTCoq. In the future, we plan to substitute the old one by this, but currently we would like to have both versions at the same time.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1044,6 +1044,13 @@ plugin:ci-vscoq:
 plugin:ci-smtcoq:
   extends: .ci-template
 
+plugin:ci-smtcoq_trakt:
+  extends: .ci-template-flambda
+  needs:
+  - build:edge-flambda
+  - library:ci-trakt
+  stage: build-2
```
Comment (SkySkimmer) [inline .gitlab-ci.yml:1052]: trakt is already build-2 so this should be build-3+

File: dev/ci/ci-smtcoq_trakt.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download smtcoq_trakt
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/smtcoq"
```
Comment (SkySkimmer) [inline dev/ci/ci-smtcoq_trakt.sh:12]:
```suggestion
( cd "${CI_BUILD_DIR}/smtcoq_trakt"
```

Comment (tomaz1502) [reply dev/ci/ci-smtcoq_trakt.sh]: Thanks! Addressed.

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
green https://gitlab.inria.fr/coq/coq/-/jobs/4483282
@coqbot merge now

**Pull request merged**